content/courses/advanced-ada/chapters/numerics.rst

```diff
@@ -19,11 +19,320 @@ Modular Types
 Numeric Literals
 ----------------
 
+Classification
+~~~~~~~~~~~~~~
+
+We've already discussed basic characteristics of numeric literals in the
+:doc:`Introduction to Ada course <courses/intro-to-ada/chapters/strongly_typed_language>`.
+We've seen that there are two kinds of numeric literals in Ada: integer
+literals and real literals. They are distinguished by the absence or presence
+of a point. For example:
```

Comment:
You may want to say "radix point" instead of just "point".

Reply:
You're right, I've just changed it.